python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -70,10 +70,16 @@ def __init__(self, database=None, driver='sqlite', host=None, port=None):
         self.database = database
         self._seeing_column = 'FWHMeff'
 
-        # a dict keyed on the user interface (i.e. args to getObservationMetaData)
-        # names of OpSim data columns.  Returns a tuple that is the
+        # a dict keyed on the user interface names of the OpSimdata columns
+        # (i.e. the args to getObservationMetaData).  Returns a tuple that is the
         # (name of data column in OpSim, transformation to go from user interface to OpSim units,
         # dtype in OpSim)
+        #
+        # Note: this dict will contain entries for every column (except propID) in the OpSim
+        # summary table, not just those the ObservationMetaDataGenerator is designed to query
+        # on.  The idea is that ObservationMetaData generated by this class will carry around
+        # records of the values of all of the associated OpSim Summary columns so that users
+        # can pass those values on to PhoSim/other tools and thier own discretion.
```

Comment:
There are only 44 columns in the OpSim Summary table, so I don't think it will be a problem.